pkg/kubernetes/README.md

```diff
@@ -12,7 +12,8 @@ This sets up a single machine kubernetes master and minion:
 Now in order to support the v1beta3 API, we need to build kubernetes
 from source or use a version later than v0.10.0:
 
-    $ git clone git@github.com:GoogleCloudPlatform/kubernetes.git
+    $ sudo yum install golang
+    $ git clone git://github.com/GoogleCloudPlatform/kubernetes.git
```

Comment:
Why not leave both github variants in there? I only use ssh clone... so for copy/pasters it would be nice to choose. A quick search revealed that this is actually the only github repo link we have in the project at this time.
A compromise would be to link to the webpage https://github.com/GoogleCloudPlatform/kubernetes/ and let people pick for themselves.